screens/EventDetails/AttendeeSection.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import { StyleSheet, TouchableOpacity, View } from "react-native"
+import { Headline, Caption } from "@components/Text"
+import { ButtonStyles } from "@lib/ButtonStyle"
+import { Ionicon } from "@components/common/Icons"
+
+interface AttendeeSectionProps {
+  color: string
+  attendeeCount: number
+}
+
+const AttendeeSection = ({ color, attendeeCount }: AttendeeSectionProps) => {
+  return (
+    <TouchableOpacity style={[styles.flexRow, styles.paddingIconSection]}>
+      <View style={{ justifyContent: "center" }}>
+        <Ionicon 
+          style={[styles.iconStyling, { backgroundColor: color }]}
+          name="people"
+          color={"white"}
+        />
+      </View>
+      <View style={styles.spacing}>
+        <Headline style={styles.textColor}>{`${attendeeCount} Attending`}</Headline>
+        <Caption>View all attendees</Caption>
```

Comment:
I wonder if we should also have "join this event to see all attendees" or remove it completely unless they've joined. For privacy purposes. Not important rn though, something we can discuss in the next meeting